cmd/utils/flags.go

```diff
@@ -845,6 +845,15 @@ var (
 		Category: flags.BuilderCategory,
 	}
 
+	BuilderDiscardRevertibleTxOnErr = &cli.BoolFlag{
+		Name: "builder.discard_revertible_tx_on_error",
```

Comment:
could you update the docs in the README?

Reply:
good catch, updated